_damo_deprecated.py

```diff
@@ -14,23 +14,23 @@
 import _damo_fmt_str
 import _damon
 
-'''
+"""
 Python2 support
-'''
+"""
 
-if sys.version.startswith('2.'):
-    _damo_deprecation_notice.deprecated(feature='Python2 support of damo',
-            deadline='2023-Q2')
+if sys.version.startswith("2."):
+    _damo_deprecation_notice.deprecated(feature="Python2 support of damo", deadline="2023-Q2")
```

Comment:
I prefer 79 characters limitation on single line.  Could you please change the configuration for that?